src/ert/config/queue_config.py

```diff
@@ -62,8 +62,8 @@ def from_dict(cls, config_dict: ConfigDict) -> QueueConfig:
             if queue_system != selected_queue_system:
                 warnings.warn(
                     ConfigWarning.with_context(
-                        f"Setting '{option_name}'for {queue_system.name} but "
-                        f"{selected_queue_system.name} is selected. ",
+                        f"Setting '{option_name}' for {queue_system.name} but "
+                        f"{selected_queue_system.name} is selected.",
```

Comment:
Will this give warnings if I have valid Torque queue options but Local queue is selected? I don't think I want that, I only want warnings if I have invalid Torque queue options and local is selected.

Reply:
I think we should be validated only when the queue is chosen otherwise just give a warning that you are setting something that is not in used anyway.

Reply:
Nah, I think it must to be possible to have ert config files that can be used both on LSF and Torque without warnings.

Reply:
Now we raise warnings if different queue is set wrongly.